spec/dummy/app/assets/config/manifest.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is needed because otherwise, database calls won't work

```
bundle exec rake db:migrate

rake aborted!

Sprockets::Railtie::ManifestNeededError: Expected to find a manifest file in `app/assets/config/manifest.js`

But did not, please create this file and use it to link any assets that need

to be rendered by your app:



Example:

  //= link_tree ../images
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css

and restart your server
```